Yoast/wordpress-seo
# Settings pages responsive layout

See #6319

Settings pages responsive layout, first pass (help center not included). The main goal, for now, is to make things functional. Refinements can be added in future iterations. Needs extensive testing on small screens.
Todo: discuss and make a decision about the banners layout when in the responsive view.

## Relevant technical choices:

As first step, some New Year's cleaning. Removed from the CSS some rules that seem no more used, most notably:
- rules related to `.yoastbox`
- `.wpseo-remove-ads`
- rules related to `.wpseotab .extension` (old extensions page)

Tried to group logically related rules together. Added inline comments and "todos".

Introduced a new media query breakpoint at `max-width: 1024px` where the CSS table layout switches to a single column layout, moving the banners sidebar at the bottom of the page. Removed the `min-width: 800px` from the content column.
Adjusted many rules related to form elements, they need special attention when testing.

## Test instructions

- build the CSS
- activate some add-ons, e.g. News, Video...
- test all the Settings pages at different viewport widths and preferably on different desktop/mobile devices
- pay special attention to form controls: worth mentioning many checkboxes and radio buttons use the "toggle switches"
- to test native checkboxes, see for example `Tools > Import and Export > Import from other SEO plugins`
- the only native radio buttons I've found are on Premium, see `Tools > Import and Export > Import redirects` so the changes in this PR should be tested on Premium too
- add-ons settings pages need to be tested too, I've tested with News, Video, and Local (Local needs a couple fixes on the PHP side, will open a specific issue)

Fixes #6384
